manual/manual/tutorials/coreexamples.etex

```diff
@@ -626,18 +634,21 @@ The source code must be put in a file with extension ".ml". It
 consists of a sequence of phrases, which will be evaluated at runtime
 in their order of appearance in the source file. Unlike in interactive
 mode, types and values are not printed automatically; the program must
-call printing functions explicitly to produce some output. Here is a
+call printing functions explicitly to produce some output.  The ";;" used
+in the interactive examples is usually not needed in
+source files created for use with the OCaml compiler.
+Here is a
 sample standalone program to print Fibonacci numbers:
 \begin{verbatim}
 (* File fib.ml *)
 let rec fib n =
-  if n < 2 then 1 else fib (n-1) + fib (n-2);;
+  if n < 2 then 1 else fib (n-1) + fib (n-2)
 let main () =
   let arg = int_of_string Sys.argv.(1) in
   print_int (fib arg);
   print_newline ();
-  exit 0;;
-main ();;
+  exit 0
+let () = main ()
 \end{verbatim}
```

Comment:
The `let () = ...` trick is idiomatic but not intuitive at all.  I see newcomers shaking their heads here.